pyproject.toml

```diff
@@ -32,6 +32,8 @@ langchain_openai = "*"
 toml = ">=0.10.2"
 pyperclip = "^1.8.2"
 langchain-anthropic = "^0.1.1"
+datasets = "^2.17.1"
+regex = "^2023.12.25"
```

Comment:
it would be nice if you can comment what functionality makes the new dependencies necessary